src/adap-b01/names/Name.ts

```diff
@@ -10,6 +10,7 @@ export class Name {
         throw new Error("needs implementation");
     }
 
+    // Returns human-readable representation of Name instance
```

Comment:
It's highly recommended to use the [jsdoc](https://jsdoc.app/) standard for code documentation.
```suggestion
    /** Returns human-readable representation of Name instance */
```